libnmstate/schema.py

```diff
@@ -297,3 +297,8 @@ class OVSBridge(object):
     PORT_NAME = 'name'
     PORT_VLAN_MODE = 'vlan-mode'
     PORT_ACCESS_TAG = 'access-tag'
+
+    class Port(object):
```

Comment:
As I remember we discussed that the should use the linux-bridge vlan abstraction which we did a few weeks ago. See line 229. It needs to come with the schema change.
Lets do it in a separate PR.

Reply:
I do not recall the discussion and I am not sure what you mean. If you like, you can adjust this PR tomorrow and I can approve it. If the change for LinuxBridge.STP is ok, I can adjust this PR to only contain the respective commits.

Reply:
Yes, the STP looks good, please extract the rest so we can take it. I mean, from this commit forward.
I will try to take over the OVS port/vlan change.

Reply:
removed the OVSBridge commits